pallets/staking-rewards/src/lib.rs

```diff
@@ -438,7 +461,7 @@ impl<T: Config> OnUnbalanced<NegativeImbalanceOf<T>> for OffsetPool<T> {
         // Should resolve into existing but resolving with creation is a safer bet anyway
         T::Currency::resolve_creating(&Pallet::<T>::account_id(), amount);
 
-        Pallet::deposit_event(Event::<T>::Deposited {
+        Pallet::deposit_event(Event::<T>::Minted {
```

Comment:
If I understood the rationale behind having another event (`Minted`), the latter should indicate the imbalance has been created inside this pallet. But in this case the imbalance is being passed in from outside. In my opinion `Deposited` is a closer match.